frontend/src/views/funding/FundingOverviewView.vue

```diff
@@ -318,15 +306,15 @@ export default {
     async loadOrganizationUsers() {
       try {
         this.loading = true
-        this.orgUsers = await OrganizationService.getUserPermissionsFacilities(this.userInfo.organizationId)
+        this.orgUsers = await UserService.getOrganizationUsers(this.userInfo.organizationId)
```

Comment:
seem likes this one is unused?

Reply:
Yeah, sorry. Change was in progress and this wasn't removed.